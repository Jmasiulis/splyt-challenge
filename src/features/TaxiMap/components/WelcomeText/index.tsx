import React from 'react';

import StyledWelcomeText from './styles';

export default function WelcomeText(): React.ReactElement {
  return (
    <StyledWelcomeText>
      Welcome! Please select how many nearby drivers you want to see.
    </StyledWelcomeText>
  );
}
