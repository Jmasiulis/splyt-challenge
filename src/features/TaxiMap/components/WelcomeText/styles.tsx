import styled from 'styled-components';

const StyledWelcomeText = styled.p`
  margin: 10px;
`;

export default StyledWelcomeText;
