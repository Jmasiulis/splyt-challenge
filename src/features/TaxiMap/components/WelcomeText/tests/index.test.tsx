import React from 'react';
import { shallow } from 'enzyme';

import WelcomeText from '..';

describe('<WelcomeText />', () => {
  it('should render', () => {
    const wrapper = shallow(<WelcomeText />);
    expect(wrapper).toMatchSnapshot();
  });
});
