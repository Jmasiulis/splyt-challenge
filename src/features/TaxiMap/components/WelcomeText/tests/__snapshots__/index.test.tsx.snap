// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomeText /> should render 1`] = `
<styled.p>
  Welcome! Please select how many nearby drivers you want to see.
</styled.p>
`;
